src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(./fonts/Roboto-Medium.ttf);
+}
+
+.body {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+}
+
+.nav__list {
+  list-style-type: none;
+  display: flex;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
```

Comment:
your `Apple` item doesn't need it

<img width="83" alt="image" src="https://user-images.githubusercontent.com/55802630/172862221-4f90c5a9-3dc5-4caa-942f-69890b4922f7.png">